### PR TITLE
Updating company direction with 2020 strategy

### PR DESCRIPTION
**Pull request by christinaforney**
From [sourcegraph product vision and strategy google doc](https://docs.google.com/document/d/1YovqooliamhrljqyiBJ_Uq4CeFH_6hqscmTjuMDT5dc/edit#).

### CODE REVIEW

**Comment by keegancsmith** [comment]
> -   Add more value inside of code hosts where developers are spending a lot of their time on issues and pull requests.
> 
> &#x2026;
> 
> -   Make the product more powerful so that developers can find things in Sourcegraph they are incapable of finding any other way.

Do we have some concrete ideas/examples here? I like the strategy, I'm just missing some more concrete ideas to help guide my developer brain for 2020 :)

A general comment. I felt a very slight unease that things were motivated by "increasing use of Sourcegraph" rather than increasing the value we give to developers. I know we all want to do the latter, but an outsider can be more pessimistic about that.

**Comment by christinaforney** [comment]
> Do we have some concrete ideas/examples here? I like the strategy, I'm just missing some more concrete ideas to help guide my developer brain for 2020 :)

Yes! This is going to be in an update to the roadmap after the strategy is finalized. (Consider this strategy the bridge (how we're going to get there) between our vision (where we want to be) and the roadmap (what we need to do))
 
> A general comment. I felt a very slight unease that things were motivated by "increasing use of Sourcegraph" rather than increasing the value we give to developers. I know we all want to do the latter, but an outsider can be more pessimistic about that.

Absolutely! Thanks for this feedback. I'm still working on wordsmithing this, and will address this point directly.

**Comment by ebrodymoore** [comment]
Absolutely love the `Build Sourcegraph into the development workflow` section. From a high level it captures everything I've been thinking about from a growth side the past couple weeks so glad to see it here 😄

File: direction/index.md [hunk 1]
```diff
@@ -1,42 +1,60 @@
...
+Sourcegraph has information that can provide immense insight for development teams and engineering leaders. Software development executives want to understand what is going on with their large teams, the projects they are investing in, and the impact developers and teams have inside of their organization. The insight needs of developers and executives is symbiotic - the information that developers use increases the quality of the insights that can be provided to leadership.
+
+#### What we want to achieve
+
+Sourcegraph is the place engineering leaders turn to to understand what is going on in their organization. A person who is not writing code everyday can use Sourcegraph for higher level insights.
+
+#### How we're going to do this
+
+- By doing research to understand what questions leadership wants to answer with insights into their organization's code.
+- We will fill more out here once initial research has been done and we have a clearer picture of the path forward.
```
Comment (nicksnyder) [inline direction/index.md:60]:
```suggestion
- Add features to Sourcegraph that provide insights that are valuable to both leaders and developers.
```

File: direction/index.md [hunk 2]
```diff
@@ -1,42 +1,60 @@
...
+
+Sourcegraph has information that can provide immense insight for development teams and engineering leaders. Software development executives want to understand what is going on with their large teams, the projects they are investing in, and the impact developers and teams have inside of their organization. The insight needs of developers and executives is symbiotic - the information that developers use increases the quality of the insights that can be provided to leadership.
+
+#### What we want to achieve
+
+Sourcegraph is the place engineering leaders turn to to understand what is going on in their organization. A person who is not writing code everyday can use Sourcegraph for higher level insights.
+
+#### How we're going to do this
+
+- By doing research to understand what questions leadership wants to answer with insights into their organization's code.
```
Comment (nicksnyder) [inline direction/index.md:59]:
```suggestion
- Do research to understand what questions engineering leaders want to answer about their organization's code.
```

File: direction/index.md (hunk 1 above)
Comment (nicksnyder) [reply direction/index.md:60]: Could add tentative examples here from [RFC 138](https://docs.google.com/document/d/1EHzor6I1GhVVIpl70mH-c10b1tNEl_p1xRMJ9qHQfoc/edit#heading=h.nhf2zflpms7n) (e.g., library impact, code coverage, code health)

File: direction/index.md [hunk 3]
```diff
@@ -1,42 +1,60 @@
...
+#### What we want to achieve
 
-Use Sourcegraph to [run large-scale code change campaigns](https://about.sourcegraph.com/product/code-change-management) to remove legacy code, fix critical security issues, and pay down tech debt. You can create campaigns across thousands of repositories and code owners. Sourcegraph automatically creates and updates all of the branches and pull requests, and you can track progress and activity in one place.
+At any organization, Sourcegraph is how developers are able to work with large and complex development workflows. Sourcegraph sustainably supports configuration of unique infrastructure and code setups.
+
+#### How we're going to do this
+
+- Solve use cases that are present at many large enterprises.
+- Reduce friction in getting Sourcegraph deployed at scale.
+- More integrated support for non-Git version control systems.
```
Comment (unknwon) [inline direction/index.md:45]: Is this true? I can't recall where exactly but I remember I saw on Slack that Git is the only version control system we would put effort on.

Just pointing out, and I have no problem with this statement.

File: direction/index.md [hunk 4]
```diff
@@ -1,42 +1,60 @@
...
+
+### Surface insight into code and development (symbiotic)
+
+#### Challenge to overcome
+
+Sourcegraph has information that can provide immense insight for development teams and engineering leaders. Software development executives want to understand what is going on with their large teams, the projects they are investing in, and the impact developers and teams have inside of their organization. The insight needs of developers and executives is symbiotic - the information that developers use increases the quality of the insights that can be provided to leadership.
+
+#### What we want to achieve
+
+Sourcegraph is the place engineering leaders turn to to understand what is going on in their organization. A person who is not writing code everyday can use Sourcegraph for higher level insights.
```
Comment (unknwon) [inline direction/index.md:55]:
```suggestion
Sourcegraph is the place engineering leaders turn to understand what is going on in their organization. A person who is not writing code everyday can use Sourcegraph for higher level insights.
```

File: direction/index.md (hunk 3 above)
Comment (christinaforney) [reply direction/index.md:45]: Good question! Over the past quarter we have gotten very strong signal from customers and prospects that supporting non-Git is important. We want Sourcegraph to be how _every_ developer finds code, not just developers using Git. 

We don't need to make every other VCS as good as the support we have for Git (at least not this year), but it does need to be far better than it is now. It should feel like an integrated experience of using Sourcegraph, not a workaround or bandaid.

File: direction/index.md [hunk 5]
```diff
@@ -1,42 +1,60 @@
...
+The investments we need to make to reach our [1 year vision](#).
 
-Want to help us achieve these goals? [We're hiring!](https://github.com/sourcegraph/careers/blob/master/job-descriptions/software-engineer.md)
+### Build Sourcegraph into the development workflow
 
-## Overview
+#### Challenge to overcome
 
-We're continually improving Sourcegraph's core features for developers:
+We believe the [natural frequency of usage](https://amplitude.com/blog/2016/10/11/product-usage-interval) (how often people use Sourcegraph) as a key part of every developer’s workflow is many times per day. Additionally, to reach our [3 year vision](#) of proactively surfacing information about code, we need to be more deeply integrated into the daily workflow, and need to have all the context about code.
```
[the remark this answers is not in the document]
Comment (christinaforney) [reply direction/index.md]: Updated, let me know if the change resolves this for you.

File: direction/index.md [hunk 6]
```diff
@@ -1,42 +1,60 @@
...
+
+- Solve use cases that are present at many large enterprises.
+- Reduce friction in getting Sourcegraph deployed at scale.
+- More integrated support for non-Git version control systems.
+
+### Surface insight into code and development (symbiotic)
+
+#### Challenge to overcome
+
+Sourcegraph has information that can provide immense insight for development teams and engineering leaders. Software development executives want to understand what is going on with their large teams, the projects they are investing in, and the impact developers and teams have inside of their organization. The insight needs of developers and executives is symbiotic - the information that developers use increases the quality of the insights that can be provided to leadership.
```
[the remark this answers is not in the document]
Comment (christinaforney) [reply direction/index.md]: Updated, let me know if you think this captures it well enough!

File: direction/index.md [hunk 7]
```diff
@@ -1,42 +1,60 @@
...
+
+### Surface insights into code and development
+
+#### Challenge to overcome
+
+Software development executives have a hard time understanding what is going on with their large teams, the projects they are investing in, and the impact developers and teams have inside of their organization. The insights and information they need is hard to come by and is not solved by other tools or services because it relies on data the development teams produce to be tied to higher order concerns.
+
+#### What we want to achieve
+
+Sourcegraph has access to the low level data needed to build insights that are high quality and meaningful. Sourcegraph feeds the symbiotic relationship between developer produced data and executive insights, creating a cyclical feedback loop of immense value. Sourcegraph is the place engineering leaders turn to understand what is going on in their organization.
```
Comment (lguychard) [inline direction/index.md:55]:
```suggestion
Sourcegraph has access to the low level data needed to build insights that are high quality and meaningful. Sourcegraph feeds the symbiotic relationship between developer produced data and executive insights, creating a cyclical feedback loop of immense value. Engineering leaders turn to Sourcegraph to understand what is going on in their organization.
```

I think the original is missing something here (phrasal verb "turn to" + non-finite "to understand")?

**Pull request merged**